presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -101,19 +108,22 @@ Date and Time Functions
 
     Returns a timestamp with time zone from ``timestamp`` and ``zone``.
 
-.. function:: from_unixtime(unixtime) -> timestamp
+.. function:: from_unixtime(unixtime) -> timestamp(3)
 
-    Returns the UNIX timestamp ``unixtime`` as a timestamp. ``unixtime`` is the number of seconds since ``1970-01-01 00:00:00``.
+    Returns the UNIX timestamp ``unixtime`` as a ``timestamp(3)``. ``unixtime``
+    is the number of seconds since ``1970-01-01 00:00:00 UTC``.
 
 .. function:: from_unixtime(unixtime, zone) -> timestamp with time zone
 
-    Returns the UNIX timestamp ``unixtime`` as a timestamp with time zone
-    using ``zone`` for the time zone. ``unixtime`` is the number of seconds since ``1970-01-01 00:00:00``.
+    Returns the UNIX timestamp ``unixtime``, a double as a timestamp with
+    time zone using ``zone`` for the time zone. ``unixtime`` is the number of
+    seconds since ``1970-01-01 00:00:00 UTC`` in ``double`` data type.
 
 .. function:: from_unixtime(unixtime, hours, minutes) -> timestamp with time zone
 
     Returns the UNIX timestamp ``unixtime`` as a timestamp with time zone
-    using ``hours`` and ``minutes`` for the time zone offset. ``unixtime`` is the number of seconds since ``1970-01-01 00:00:00``.
+    using ``hours`` and ``minutes`` for the time zone offset. ``unixtime`` is
```

Comment:
Same

Reply:
this add the data type as requested by @findepi

Reply:
I am out of context. what did i request that @electrum does not enjoy?

Reply:
Please revert this change as it's unrelated to timestamp precision.